bin/find_scheme.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+
+xcodebuild -list -project $@ \
```

Comment:
I think `$@` should be in quotes: `"$@"`, which correctly passes through arguments with spaces like `find_scheme "hello there" "hey"` rather than turning it into three arguments.